scripts/test-ocs-ci.sh

```diff
@@ -53,6 +53,11 @@ pushd ../src/ocs-ci
 
 source $WORKSPACE/venv/bin/activate		# enter 'deactivate' in venv shell to exit
 
+# set bastion ip in the ocs-ci-conf.yaml
+if [ -n "$BASTION_IP" ]; then
```

Comment:
This code change should be in the function update_supplemental_ocsci_config() in scripts/helper/parameters.sh.

Reply:
Luke, Thanks for the review.  I amended this PR.